packages/web-component/src/index.ds.ts

```diff
@@ -0,0 +1,21 @@
+export interface Page {
```

Comment:
Why are these moved to this location?

Reply:
This is some refactoring Del was doing. I don't know all the details. It makes more sense with the rest of the page_transition PR.